src/olympia/scanners/views.py

```diff
@@ -61,7 +61,13 @@ def get_queryset(self):
             .all()
         )
         bad_results = (
-            bad_results.filter(state=TRUE_POSITIVE)
+            bad_results.filter(
+                Q(state=TRUE_POSITIVE) | Q(
+                    version__versionlog__activity_log__action__in=(
+                        amo.LOG.BLOCKLIST_BLOCK_ADDED.id,
```

Comment:
I think you can add `amo.LOG.BLOCKLIST_BLOCK_EDITED` too.

Reply:
added, thanks